draco/asp/soft.lp

```diff
@@ -0,0 +1,4 @@
+% ====== Preferences ======
+% @soft(aggregate) Prefer to use raw (no aggregate).
+violation(aggregate,E) :-
```

Comment:
This will not work entirely since we are forbidding violations right now. Should we add a list of violations that are forbidden or have a second type of violation that's for soft constraints? What do you think?

Reply:
Can we treat hard and soft equally, remove the forbidding violation requirement and assign hard with infinite weight?

Reply:
We discussed in a a meeting that while it's possible, it's not advisable.